src/components/Firestore/DocumentEditor/types.ts

```diff
@@ -46,7 +46,7 @@ export type Store = {
 export function isPrimitive(
   data: FirestoreAny | DocumentPath
 ): data is PrimitiveValue {
-  return isNativePrimitive(data) || data instanceof DocumentPath;
+  return (isNativePrimitive(data) as any) || data instanceof DocumentPath;
```

Comment:
Is this really needed? It looks like `isNativePrimitive` has a return type of boolean, no?

Reply:
weird - I removed it (which makes sense) - I'm not sure why the type-error came up before. Reverted.